requirements.txt

```diff
@@ -24,4 +24,5 @@ seaborn
 azure-storage-blob
 azure-identity
 einops
-nltk
\ No newline at end of file
+nltk
```

Comment:
where is this used ?

Reply:
addressed

Reply:
its used by rouge evaluators (not automatically installed dependency)